src/map/lapack2flash/FLASH_lauum.c

```diff
@@ -42,6 +42,9 @@
   dim_t        blocksize = min( FLASH_get_preferred_blocksize(),\
                                 *ldim_A );                      \
                                                                 \
+  FLA_Bool toggle = FLASH_Check_Offload_to_GPU( blocksize, *n,  \
+                          *n, FLASH_get_tile_offload() );       \
```

Comment:
*n is passed for both m and n when used for both in FLASH_Obj_create_without_buffer()